### PR TITLE
PRIME-557 Auto Adjudicate RN LPN RPN + fix backend tests

### PR DESCRIPTION
**Pull request by AnaisHebert**

### CODE REVIEW

File: prime-dotnet-webapi-tests/Mocks/EmailServiceMock.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public override void SeedData()
             // no data to seed, as it is done in the base class
         }
 
-        public Task SendProvisionerLinkAsync(string provisionerEmail, EnrolmentCertificateAccessToken token, string ccEmail)
+        public Task SendProvisionerLinkAsync(string provsionerName, string provisionerEmail, EnrolmentCertificateAccessToken token, string ccEmail)
```
Comment (mtpultz) [inline prime-dotnet-webapi-tests/Mocks/EmailServiceMock.cs]: Fnoud a tpoy on provisionerName

File: prime-dotnet-webapi/Migrations/20200319180139_ValidateLicenceFlag.cs [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+namespace Prime.Migrations
+{
+    public partial class ValidateLicenceFlag : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(
+                table: "DefaultPrivilege",
+                keyColumns: new[] { "PrivilegeId", "LicenseCode" },
+                keyValues: new object[] { 18, 16 });
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Migrations/20200319180139_ValidateLicenceFlag.cs]: Could you make this an UpdateData rather than a drop and re-add?

Comment (AnaisHebert) [reply prime-dotnet-webapi/Migrations/20200319180139_ValidateLicenceFlag.cs]: It is removing one privilege and replacing it with another so i dont think it needs to be an update in this case.

File: prime-dotnet-webapi/Services/AutomaticAdjudicationService.cs [hunk 3]
```diff
@@ -134,38 +134,41 @@ protected override async Task<bool> ProcessRuleInternal(Enrollee enrollee)
 
                 foreach (var cert in enrollee.Certifications)
                 {
-                    PharmanetCollegeRecord record = null;
-                    try
+                    if (cert.License.Validate)
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Services/AutomaticAdjudicationService.cs]: Could you please flip the logic and make this
`if(!cert.Licencse.Validate)
{
    continue;
}`
it makes it easier to read in my opinion

Comment (james-hollinger) [reply prime-dotnet-webapi/Services/AutomaticAdjudicationService.cs]: or, alternatively, you could use
`foreach (var cert in enrollee.Certifications.Where(c => c.License.Validate))`

Comment (AnaisHebert) [reply prime-dotnet-webapi/Services/AutomaticAdjudicationService.cs]: > or, alternatively, you could use
> `foreach (var cert in enrollee.Certifications.Where(c => c.License.Validate))`

I like this better the other ways feels awkward to me

**Pull request merged**